sdk/src/main/java/com/deploygate/sdk/DeployGate.java

```diff
@@ -68,6 +66,9 @@ public class DeployGate {
     private final ILogcatInstructionSerializer mLogcatInstructionSerializer;
     private final CustomLogInstructionSerializer mCustomLogInstructionSerializer;
     private final HashSet<DeployGateCallback> mCallbacks;
+    private final HashSet<DeployGateInitializeCallback> mInitializeCallbacks;
+    private final HashSet<DeployGateStatusChangeCallback> mStatusChangeCallbacks;
+    private final HashSet<DeployGateUpdateAvailableCallback> mUpdateAvailableCallbacks;
```

Comment:
I chose the implementation to keep `mCallbacks` to minimize the effect on existing code.
It will be deleted with DeployGateCallback.

Reply:
I focused on minimizing _code changes_.
I understood it is important to minimize _behavior changes_ in this case. 

I'll fix it.